src/util/pagination.js

```diff
@@ -1,3 +1,33 @@
+const _ = require('lodash')
+const createError = require('http-errors')
+const { createCursor, parseCursor } = require('./cursor')
+
+const reverseOrder = (order) => order === 'asc' ? 'desc' : 'asc'
+const reverseOperator = (operator) => {
+  // reverse the order, not negate the operator
+  // negation of '>' is '<='
```

Comment:
You could add parentheses to this comment to make it clear it’s an irrelevant example.